contracts/ERC677BridgeToken.sol

```diff
@@ -24,7 +26,7 @@ contract ERC677BridgeToken is
         string _name,
         string _symbol,
         uint8 _decimals)
-    public DetailedERC20(_name, _symbol, _decimals) {}
+    public ERC20Detailed(_name, _symbol, _decimals) {}
 
     function setBridgeContract(address _bridgeContract) onlyOwner public {
```

Comment:
The token owner can change bridge contract. Changing the bridge to other contract will disallow the usage of  the bridge.

Reply:
owner should not be allowed to change the bridge, or it is the bridge itself.

Reply:
Makes sense. I make `setBridgeContract` with `onlyMinter` modifier.  The minter is the bridge. This makes the permissions similar to the previous case. Before it was `onlyOwner` when the bridge was the owner.